include/z64.h

```diff
@@ -283,8 +283,8 @@ typedef struct {
 
 typedef struct {
     /* 0x00 */ u16 countdown;
-    /* 0x04 */ Vec3f originPos;
-    /* 0x10 */ Vec3f relativePos;
+    /* 0x04 */ Vec3f originWorldPos;
+    /* 0x10 */ Vec3f projectedPos;
```

Comment:
Why is one called "origin"?

Imo if it's like "sound origin" then that's already implied by being in the `SoundSource` struct,
and if it's like "base value for projected" then "world" is enough already

Reply:
I included origin just to be explicitly clear as to what this position is for easier reading from someone new who's trying to learn what "soundSource" means. If "worldPos" is enough to imply that it's the original world position, then I'm fine with removing it. I think I'll still leave an explicit comment describing that it's an original position in the z64.h struct, I think that should be fine